projectlicense.md

```diff
@@ -1,61 +1,51 @@
 # Licensing terms
 
-This project is licensed under the terms of the Modified BSD License
-(also known as New or Revised or 3-Clause BSD), as follows:
-
-- Copyright (c) 2001-2015, IPython Development Team
-- Copyright (c) 2015-, Jupyter Development Team
```

Comment:
does this deletion retroactively change the copyright that used to be there? I would have thought that any previous BSD license copyrights would remain in place, but maybe it's not a big deal here.

Reply:
I think probably `Jupyter Development Team` can be (optionally?) changed to `Project Jupyter Contributors`. I think any other copyrights probably ought to be left in place, and the `Project Jupyter Contributors` copyright should be appended.

Reply:
>  I think any other copyrights probably ought to be left in place, and the Project Jupyter Contributors copyright should be appended.

Makes sense to me.